src/bin/cargo-flamegraph.rs

```diff
@@ -84,8 +88,10 @@ fn build(opt: &Opt, kind: impl IntoIterator<Item = String>) -> anyhow::Result<Ve
         cmd.arg("build");
     }
 
+    if let Some(profile) = &opt.profile {
+        cmd.arg("--profile").arg(profile);
     // do not use `--release` when we are building for `bench`
```

Comment:
Not sure if this will pass `cargo fmt`, the comment placement seems a bit weird here.

Reply:
This is actually why I originally didn't collapse the `else if`.

Reply:
btw this is already has `cargo fmt` applied

Reply:
Okay, let's just move the comment inside the `else if`? That seems like a cleaner way to clarify.